ember-cli-build.js

```diff
@@ -71,7 +71,6 @@ module.exports = function(defaults) {
 
     app.import('node_modules/dropzone/dist/dropzone.css');
     app.import('node_modules/dropzone/dist/dropzone.js');
-    app.import('node_modules/osf-style/css/base.css');
     app.import('node_modules/@centerforopenscience/osf-style/css/base.css');
```

Comment:
I don't think we need this since we are already importing the scss from osf-style.

Reply:
Just removed and rebuilt to make sure. Confirmed we don't need it (and it's redundant).